extension.js

```diff
@@ -2,10 +2,21 @@ const Main = imports.ui.main;
 const Meta = imports.gi.Meta;
 const Shell = imports.gi.Shell;
 const Me = imports.misc.extensionUtils.getCurrentExtension();
-const {spawnCommandLine, spawn} = imports.misc.util;
+const { spawnCommandLine, spawn } = imports.misc.util;
```

Comment:
I think lsp auto formatter had kicked in without me knowing (i don't do much js), i'll remove them if needed.

Reply:
Yes please, remove them. I'm using ESLint in PyCharm that would tend to restore them back.